data/image_train_item.py

```diff
@@ -245,7 +247,7 @@ def __compute_target_width_height(self):
         self.target_wh = None
         try:
             with PIL.Image.open(self.pathname) as image:
-                image = self._try_transpose(image, print_error=True).convert('RGB')
```

Comment:
convert to RGB unneeded for compute target wh, but it also doesn't seem to have much to do with performance